### PR TITLE
Add const Inplace& argument to Summary::eval()

### PR DESCRIPTION
**Pull request by joakim-hove**

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this opm-simulators=2935 please

File: src/opm/output/eclipse/Summary.cpp [hunk 1]
```diff
@@ -2780,6 +2782,7 @@ eval(const int                          sim_step,
      const data::WellRates&             well_solution,
      const data::GroupAndNetworkValues& grp_nwrk_solution,
      GlobalProcessParameters&           single_values,
+     const Inplace&                     initial_inplace,
```
Comment (bska) [inline src/opm/output/eclipse/Summary.cpp:2785]: This parameter is currently unused and will produce warnings.  Unless there is a follow-up PR that uses the values coming soon, I suggest you leave the object unnamed.

Comment (joakim-hove) [reply src/opm/output/eclipse/Summary.cpp:2785]: It will be used in this PR.

**Comment by joakim-hove** [comment]
OK - a bit back on forth here (as usual ...). This PR is now *only* the API change - the real change comes here: #2124

**Comment by joakim-hove** [comment]
jenkins build this opm-simulators=2935 please

**Comment by joakim-hove** [comment]
jenkins build this opm-simulators=2935 please

**Pull request merged**